src/analysis/ffi_type.rs

```diff
@@ -127,8 +125,8 @@ fn ffi_inner(env: &Env, tid: TypeId, inner: &str) -> Result {
             }
             fix_name(env, tid, inner)
         }
-        Type::CArray(inner_tid) => ffi_inner(env, inner_tid, inner),
-        Type::FixedArray(inner_tid, size, _) => ffi_inner(env, inner_tid, inner)
+        &Type::CArray(inner_tid) => ffi_inner(env, inner_tid, inner),
+        &Type::FixedArray(inner_tid, size, _) => ffi_inner(env, inner_tid, inner)
```

Comment:
That's ugly :) Does this maybe want a `Type::FixedArray(ref inner_tid, size, _) => ...` or so?

Reply:
That's the other way around :) - realistically we need `Type::FixedArray(&inner_tid, &size, _)` to destructure the borrow on _just_ those two fields.